src/test/java/com/falco/workshop/validation/ValidationTest.java

```diff
@@ -32,7 +43,7 @@ public void shouldDetectOverlappingRows() {
     }
 
     private LocalDateTime date(String from) {
-        return LocalDateTime.parse(from.replaceAll(" ","T"));
+        return LocalDateTime.parse(from.replaceAll(" ", "T"));
```

Comment:
Maybe LocalDateTime.parse(from, DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")) would be more straight forward?